yoda/utils/sambanova_endpoint.py

```diff
@@ -0,0 +1,205 @@
+"""Wrapper around Sambanova APIs."""
```

Comment:
Let's use the file already exisitng in the starter kit. No need to have it specifically fot yoda.